subsys/bluetooth/host/hci_raw.c

```diff
@@ -51,11 +69,86 @@ struct net_buf *bt_buf_get_rx(enum bt_buf_type type, s32_t timeout)
 {
 	struct net_buf *buf;
 
+	switch (type) {
+	case BT_BUF_EVT:
+	case BT_BUF_ACL_IN:
+		break;
+	default:
+		BT_ERR("Invalid type: %u", type);
+		return NULL;
```

Comment:
Wouldn't you get to this "Invalid type" branch now that you added `BT_BUF_H4`?

Reply:
Actually BT_BUF_H4 shall not be passed to bt_buf_get_rx since this is used for data originating from the stack itself, so it wont have H:4 header which is actually added later in the function.

Reply:
Sorry, I thought I was looking at `get_tx()`